torchtext/data/datasets_utils.py

```diff
@@ -213,7 +213,7 @@ def _wrap_split_argument_with_fn(fn, splits):
         raise ValueError("Internal Error: Given function {} did not adhere to standard signature.".format(fn))
 
     @functools.wraps(fn)
-    def new_fn(root='.data', split=splits, **kwargs):
+    def new_fn(root=os.path.expanduser('~/.torchtext/cache'), split=splits, **kwargs):
```

Comment:
Is this BC-breaking change?

Reply:
It is a good question. This is the default path where datasets will be downloaded. In worst case, the data would be downloaded again for users relying on default behavior instead of providing their own root folder. Any think else that could potentially make it BC-breaking?

Reply:
I see. I think it's on a boarder-line but it is okay to it as backward compatible.
Yet, we probably should emphasize in the upcoming release note.